packages/elements/src/calendar/__test__/calendar.defaults.test.js

```diff
@@ -147,4 +164,79 @@ describe('calendar/Defaults', function () {
       await expect(el).shadowDom.to.equalSnapshot();
     });
   });
+
+  describe('before-cell-render event fires correctly', function () {
+    it('before-cell-render event fires on first render', async function () {
```

Comment:
I would personally prefer using `it` follows by verb option or even use `describe` to further simplify repetitive test case namings.
```suggestion
    it('fires before-cell-render on first render', async function () {
```

or
```suggestion
describe('before-cell-render event, function () {
    it('fires on first render', async function () {
```

Reply:
thanks! Updated to "should fire ..." pattern.